language-design.md

```diff
@@ -16,19 +16,78 @@ are:
 For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
-
-
+> "These stereotypes come about because programmers confuse their strong views about 
+> languages with their views about the users of the languages. Consider, for instance, 
+> this anonymous Reddit comment: “Node.js is not popular because of its non-blocking features. 
+> It is popular because now dumb javascript devs can write server code. Earlier they had to 
+> learn real languages like Perl/Java/Python etc. to do that.
+> Our commenter assumes that using JavaScript, as opposed to a “real” language, is a sign of 
+> incapacity. But in technical terms, JavaScript, Perl, and Python are fairly similar (they are
+> all interpreted, dynamically typed, multi-paradigm languages). The only difference is that 
+> our commenter has ideas about who uses each language–and about the languages that “real 
+> programmers” use"
+[Yang and Rabkin, 2015].
+
+This quote is quite accurate in both of our experiences. People stereotype languages based on the users.
+For example the article mentions that Python and Ruby are considered "girly." Both of
+these languages are used a lot in web development, which tends to have a large UI and design
+component. This strong association with creativity, a steretypically feminine quality, means that the languages
+are stereotyped as "girly." On the other hand, C or C++ are seen as manly are often used to solve back-end 
```

Comment:
I agree with your analysis here. Because of the associations of masculinity and femininity to languages like C++ and Python, we see cliques form amongst users of the languages. When talking about the gendered differentiation that hurts everyone, it would've been good to explain in what ways does this hurt people. Especially because you mention that this is accurate to both your experiences, adding your experiences or some other ways the differentiation could hurt people would have been interesting.